diagnostics/model_accuracy/verify_inference_model.py

```diff
@@ -0,0 +1,144 @@
+            cmd = "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0 python eval_image_classifier.py --alsologtostderr --checkpoint_path=" + CHECKPOINT + " --dataset_dir=" + DATASET_DIR + " --dataset_name=imagenet --dataset_split_name=validation --model_name=mobilenet_v1"
+            p = command_executor(cmd)
+            os.chdir(pwd)
+
+        if (model_name in "ResNet50_v1"):
+            cmd = "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0 python eval_image_classifier.py --alsologtostderr --checkpoint_path=" + CHECKPOINT + " --dataset_dir=" + DATASET_DIR + " --dataset_name=imagenet --dataset_split_name=validation --model_name=resnet_v1_50 --labels_offset=1"
+            p = command_executor(cmd)
+            os.chdir(pwd)
+
+        #TODO:SSD-MobileNet_v1 does not run using ngraph, need to debug.
+        # if (model_name in "SSD-MobileNet_v1"):
+        #     command_executor(
+        #         "OMP_NUM_THREADS=28 KMP_AFFINITY=granularity=fine,compact,1,0 python object_detection/model_main.py --logtostderr --pipeline_config_path=object_detection/samples/configs/ssd_mobilenet_v1_coco.config --checkpoint_dir="
+        #         + CHECKPOINT + "--run_once=True")
+        #     os.chdir(pwd)
+
+        return model_name, p
+    except:
+        print(
+            "Please pass a valid Model name. Availble models are Inception_v4, MobileNet_v1, ResNet50_v1"
+        )
+        sys.exit(0)
+
+
+def check_accuracy(model, p):
+    #check if the accuracy of the model inference matches with the published numbers
+    accuracy = '[{"model_name" : "Inception_v4", "accuracy" : "0.95194"}, {"model_name" : "ResNet50_v1", "accuracy" : "0.752"}, {"model_name" : "MobileNet_v1", "accuracy" : "0.71018"}]'
+    data = json.loads(accuracy)
+
+    for line in p.splitlines():
+        print(line)
+        if ('eval/Accuracy'.encode() in line):
+            top1_accuracy = re.search("\[(.*?)\]", line.decode()).group(1)
+        #for now we just validate top 1 accuracy, but calculating top5 anyway.
+        if ('eval/Recall_5'.encode() in line):
+            top5_accuracy = float(
+                re.search("\[(.*?)\]", line.decode()).group(1))
+
+    for i, d in enumerate(data):
+        if (model in data[i]["model_name"]):
+            # Tolerance check
+            diff = abs(float(top1_accuracy) - float(data[i]["accuracy"]))
+            if (diff <= 0.001):
+                print("\nResult:Model Accuracy is expected for " + model + ' ' +
+                      top1_accuracy)
+            else:
+                print("\nResult:Model Accuracy is not as expected for " +
+                      model + ' ' + top1_accuracy)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Accuracy verification for TF models using ngraph.')
+
+    parser.add_argument('--model_name', help='Model name to run inference')
+    cwd = os.getcwd()
+    # This script must be run from this location
```

Comment:
Why? Please remove this restriction.
You can add an optional parameter specifying the source of the model repository location on disk. If not then the local directory will be used.